static/css/mozsearch.css

```diff
@@ -153,15 +153,14 @@ td#line-numbers {
   display: flex;
 }
 
-/* In order to display the line numbers and have them visible to screen readers
-   but not appear in any copy-and-pasted selection, we create a before
-   pseudo-element with generated content. */
-.line-number::before {
-    content: attr(data-line-number);
-}
 .line-number {
     display: inline-block;
     cursor: pointer;
+    -moz-user-select: none;
+    -o-user-select: none;
+    -khtml-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
```

Comment:
Probably not worth adding the -o- and khtml prefixed but fine... Also maybe add the unprefixed version?

Reply:
+1 to adding the unprefixed version

Reply:
The unprefixed version is already there.  (All of these were actually here before.  I just experienced a bout of extreme optimism and removed all the prefixed versions.)

Reply:
(And my original a11y prototyping didn't actually use our CSS, I was working with github before...)